tests/plugins/autolink/autolink.js

```diff
@@ -14,6 +14,15 @@
 				removePlugins: 'link'
 			}
 		},
+		optionalParameters: {
+			config: {
+				allowedContent: true,
+				pasteFilter: null,
+				removePlugins: 'link',
+				autolink_urlRegex: /^(https?|ftp):\/\/(-\.)?([^\s\/?\.#]+\.?){5}(\/\S*)?[^\s\.,]$/i,
+				autolink_emailRegex: /^[a-zA-Z0-9.!#$%&'*+\/=?^_`{|}~-]+@[a-zA-Z0-9](?:[a-zA-Z0-9-]{0,61}[a-zA-Z0-9])?(?:\.[a-zA-Z0-9](?:[a-zA-Z0-9-]{0,61}[a-zA-Z0-9])?)*$/
```

Comment:
These regexes are pretty complicated and as we only need to check if custom regex has been used (instead of the default one). Maybe it could be simplified to something like `/^http:\/\/foobar\.com$/`, so the expected result is crystal clear? The same for the email address.